recipes/ncbimeta/meta.yaml

```diff
@@ -17,21 +17,21 @@ build:
 requirements:
   host:
     - pip
-    - python >3
+    - python>=3.6
   run:
-    - python >3
-    - biopython >=1.74,<1.77
-    - pyyaml >=5.1.2
-    - lxml >=4.4.2
-    - numpy >=1.16.4
+    - python>=3.6
+    - biopython>=1.74,<1.77
+    - pyyaml=5.4
+    - lxml=4.6.3
+    - numpy=1.19.5
```

Comment:
Can it really only use these exact versions of pyyaml, lxml and numpy?

Reply:
Thanks for the feedback!

For lxml and pyyaml, they just need to be at least those versions (which are the most recent) so I can change it to ">=". Earlier versions of lxml and pyyaml have known security vulnerabilities of moderate to high risk and Github nags me to keep those dependencies up to date.

For numpy, there is a narrow range of versions that work for both Python 3.6 and 3.9. If you think it's worthwhile, I can test out that full range so it's not an exact version.

Reply:
I presume newer numpy versions will work in python 3.6 through 3.9, so maybe just a `>=1.19.5` there as well.

Reply:
Sadly, numpy dropped support for Python 3.6 in version 1.20, and only started releasing binary wheels for Python 3.9 in version 1.19.3. Without the binary wheels, build errors occur in my CI. I did some testing and the numpy versions that can support this range of Python are extremely narrow, 1.19.3-1.19.5 . I'll do a new commit with relaxed dependencies for your review.

This might be the sign that I'll also be dropping Python 3.6 in my next release...